tests/report_test.go

```diff
@@ -40,8 +40,8 @@ func (c *ClientTests) TestReportCustomColumnsCreate() {
 
 	// verify parameters by checking VCR cassette for correct URL
 	// Some reports take a long time to generate, so we won't be able to consistently pull the report
-	// There's unfortunately no way to check if the columns were included in the final report without parsing the CSV
-	// so we assume, if we haven't gotten an error by this point, we've made the API calls correctly
```

Comment:
Isn't this a continuation of the sentence? Uppercasing doesn't seem like the right change